test/unit/specs/data/json-update-builder.js

```diff
@@ -0,0 +1,154 @@
+import { describe, it } from 'mocha'
+
+import Backendless from '../../helpers/sandbox'
+import { expect } from 'chai'
+
+describe('<Data> JSON Update Builder', function() {
+  let jsonUpdateBuilder
+
+  beforeEach(() => {
+    jsonUpdateBuilder = Backendless.Data.JSONUpdateBuilder
```

Comment:
it should start with Uppercase letter as it is a class

Reply:
+